build.gradle

```diff
@@ -48,6 +48,12 @@ allprojects {
     repositories {
         google()
         mavenCentral()
+        maven {
+            url "https://a8c-libs.s3.amazonaws.com/android/jcenter-mirror"
+            content {
+                includeVersion "com.android.volley", "volley", "1.1.1"
```

Comment:
I had to add this to make the CI work, I'm not sure what caused the new failures 😕, looking at maven [listing](https://mvnrepository.com/artifact/com.android.volley/volley/1.1.1) of this dependency, we can see it's not available in neither the Google nor the Maven Central repositories, this line should have been needed even before 🫤.

cc @ParaskP7 any ideas.

Anyway, this PR is urgent, and this should block the merge regardless of the cause.

Reply:
👋 @hichamboushaba and thanks for the ping!

Yea, I am not 100% sure what happened there, but this seems pretty similar to this other case of such a miss (p1723103849516809/1723070238.781829-slack-C036Y8QL4). 🤔

I think that `jcenter` [[link](https://mvnrepository.com/artifact/com.android.volley/volley)] is just deleting artifacts in general, maybe more aggressively nowadays, and we just can’t download that specific one from anywhere atm (or at least from those 2 places that we know of: [[1](https://repo1.maven.org/maven2/com/android/volley/volley/)] + [[2](https://maven.google.com/web/index.html?q=volley#com.android.volley:volley)]). It  is good that we at least have there available on our own `jcenter-mirror` repo. 🍀

PS: Thanks for making it work for yourselves by adding the `jcenter-mirror` configuration, just like it was done on multiple other repos of ours. 🥇